bundles/af-core/src/test/resources/form/recaptcha/exporter-recaptcha.json

```diff
@@ -31,7 +31,9 @@
         "type": "image",
         "version": null
       }
-    }
+    },
+    "mandatory": "true",
```

Comment:
mandatory translates to required as first class prop. why is this part of custom prop?

Reply:
Did not see an instance of it set in the code way back. Later found out we have a lot of these and others via content.xml, of which I forgot to put this one back.
Adding back to reserve list now.